install/mysql/glpi-empty.sql

```diff
@@ -4038,6 +4038,8 @@ CREATE TABLE `glpi_knowbaseitems` (
   `name` text,
   `answer` longtext,
   `is_faq` tinyint NOT NULL DEFAULT '0',
+  `is_token_url` tinyint NOT NULL DEFAULT '0',
```

Comment:
Name could be improved, it doesn't seem related to the label "Share with a unique URL".

Something like this ?
```suggestion
  `allow_access_using_token` tinyint NOT NULL DEFAULT '0',
```